step-cli/step-cli-launcher/src/main/java/step/cli/StepConsole.java

```diff
@@ -407,7 +407,7 @@ protected void handleApRemoteExecuteCommand() {
                 checkStepUrlRequired();
                 checkEeOptionsConsistency(spec);
 
-                executeRemotely(stepUrl,
+checkStepControllerVersion();                executeRemotely(stepUrl,
```

Comment:
checkStepControllerVersion has been added here, maybe by a merge, I guess it should be on another line

Reply:
Yes, maybe. Now I've aligned the format here